inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -259,6 +295,36 @@ public String printConfig()
     {
         return null;
     }
+    
+    
+    /**
+     * Takes in IRI for identifier and returns teh label String
+     * Eg: 
+     * InputParameter :- http://www.w3.org/TR/2003/PR-owl-guide-20031209/wine#RoseDAnjou
+     * Returned :- "Concept"+MtasToken.DELIMITER+RoseDAnjou
+     * @param iri
+     * @return
+     */
+    public String getUILabel(String iri) {
+    	
+    	StringBuffer labelStr  = new StringBuffer("");
+    	Optional kbReference = null;
```

Comment:
Optional should have a generic type, e.g. `Optional<KBHandle>`.

Reply:
You also do not need to set it to null, just assign it directly